src/main/java/io/mosip/mimoto/util/JoseUtil.java

```diff
@@ -53,7 +53,7 @@ public static JwkDto getJwkFromPublicKey(String publicKeyString) throws NoSuchAl
     }
 
     private static RSAPublicKeySpec getKeySpecForRSAKey(String publicKeyString) {
-        String key = publicKeyString.replaceAll("\\n", "")
+        String key = publicKeyString.replace("\n", "")
```

Comment:
To resolve the critical issue -  Replace this call to "replaceAll()" by a call to the "replace()" method.
"String#replace" should be preferred to "String#replaceAll" [java:S5361](https://sonarcloud.io/organizations/mosip/rules?open=java%3AS5361&rule_key=java%3AS5361)

https://sonarcloud.io/project/issues?open=AYYNG821bizUviPiOsD7&id=mosip_mimoto

Since replace("/n") works the similar way as .replaceAll("//n"), the changes were made. This is also tested in JoseUtilTest.java